src/isIsogram.test.js

```diff
@@ -6,4 +6,20 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for an empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should return true = "playgrounds"', () => {
```

Comment:
It is better to use more informative test names.
```suggestion
  it('should return true if entered string has no repeatings', () => {
```
Check the other tests.
The equivalence classes technique suits best for this task.

Reply:
Fixed.